src/main/java/uk/gov/hmcts/reform/amapi/controllers/AmLibProxyController.java

```diff
@@ -34,19 +36,27 @@
     @SuppressWarnings("unchecked") // supressing compiler warning about casting from Object to List<String>
     @PostMapping("/create-resource-access")
     public void createResourceAccess(@RequestBody Map<String, Object> amData) {
```

Comment:
Try changing type from `Map<String, Object>` to `ExplicitAccessGrant`

Reply:
lots less code 👍